app/views/carts/_cart_row.html.erb

```diff
@@ -1,26 +1,38 @@
-<tr>
-  <td>
-    <div class="product-item">
-      <%= link_to @product do %>
-        <%= load_image_product @product %>
-      <% end %>
-      <div class="product-info">
-      <h4 class="product-title">
-        <%= link_to order.product.name, "#" %>
-      </h4>
-      <span><em><%= link_to order.product.category_name, "#" %></em></span>
-      <span><em><%= t "carts.in_stock" %></em> <%= order.product.quantity %></span>
+<div class="product_item">
+  <tr>
+    <td>
+      <div class="product-item">
+        <%= link_to @product do %>
+          <%= load_image_product @product %>
+        <% end %>
+        <div class="product-info">
+        <h4 class="product-title">
+          <%= link_to order.product.name, "#" %>
+        </h4>
+        <span><em><%= link_to order.product.category_name, "#" %></em></span>
+        <span><em><%= t "carts.in_stock" %></em> <%= order.product.quantity %></span>
```

Comment:
` order.product.quantity` em tìm hiểu về `delagate in rails ` đi nhé

Reply:
Dạ